pages/common/redis-server.md

```diff
@@ -0,0 +1,24 @@
+# redis-server
+
+> Persistent key-value database.
+> More information: <https://redis.io>.
+
+- Start Redis server, using the default port (6379), and write logs to `stdout`:
+
+`redis-server`
+
+- Start Redis server, using the default port, as a background process:
+
+`redis-server --daemonize yes`
+
+- Start Redis server, using the specified port, as a background process:
+
+`redis-server --port {{port}} --daemonize yes`
+
+- Start Redis server with a custom configuration file:
+
+`redis-server {{path/to/redis.conf}}`
+
+- Start Redis server with verbose logging:
+
+`redis-server --loglevel verbose`
```

Comment:
Are there different log levels you can choose? If so, it might be worth indicating that. :+1:
```suggestion
`redis-server --loglevel {{verbose}}`
```

Reply:
These are the options, notice is the default:
```suggestion
`redis-server --loglevel {{warning|notice|verbose|debug}}`
```

Reply:
That looks better 👍🏻